### DISCUSSION TITLE
File Search seems very limited & doesn't find most files?

### DISCUSSION THREAD

**Discussion by UnchartedSystems** [regolith-linux/regolith-desktop#896] [General]
From what I can tell, file search has a very limited scope.

- It can't find hidden files
- It can't find any files in my custom directories ,such as `/home/{user}/Code/`

Those are the major gripes that make me never reach for file search when it could be a very useful element of Regolith. I also wish there was a hotkey to copy the path of highlighted files so that I could open them in whatever program I want.

**Comment by w18g** [comment]
"file search" is a front end to gnome tracker. It is quite limited because it can also be a cpu hog when indexing files.

By default I see that the tracker only indexes the home dir and downloads dir....so add in other directories use this command:

tracker3 index -a [directory or file]
